addons/mass_mailing/tests/test_mailing_internals.py

```diff
@@ -205,6 +208,45 @@ def test_mailing_on_res_partner(self):
             mailing, partner_a | partner_b, check_mail=True
         )
 
+    @users('user_marketing')
+    @mute_logger('odoo.addons.mail.models.mail_mail')
+    def test_mailing_cron_trigger(self):
```

Comment:
Could you move it before test_mailing_on_res_partner ? Reason is that as tests are executed by alphabetical order, better have them defined in that order. Helps finding its way through the file when solving crashed tests.